Tests/KnitCodeGenTests/TypeNamerTests.swift

```diff
@@ -149,6 +149,23 @@ final class TypeNamerTests: XCTestCase {
         )
     }
 
+    func testMultipleGenerics() {
+        assertComputedIdentifier(
+            type: "AnyPublisher<Set<AppletId>, Never>",
+            expectedIdentifier: "appletIdSetPublisher"
+        )
```

Comment:
This case was the one that was causing problems, it was previously returning `set<AppletId>Publisher` which is not a valid swift identifier so the code would not compile.